compiler/src/dotty/tools/dotc/interactive/Completion.scala

```diff
@@ -524,10 +526,10 @@ object Completion:
       // There are four possible ways for an extension method to be applicable
 
       // 1. The extension method is visible under a simple name, by being defined or inherited or imported in a scope enclosing the reference.
-      val termCompleter = new Completer(Mode.Term, prefix, pos)
-      val extMethodsInScope = termCompleter.scopeCompletions.toList.flatMap:
-        case (name, denots) => denots.collect:
-          case d: SymDenotation if d.isTerm && d.termRef.symbol.is(Extension) => (d.termRef, name.asTermName)
+      val termCompleter = new Completer(Mode.Term, prefix, pos, prefixFilter)
+      val extMethodsInScope = termCompleter.scopeCompletions.toList.flatMap {
+        case (name, denots) => denots.collect { case d: SymDenotation if d.isTerm => (d.termRef, name.asTermName) }
```

Comment:
why is that `d.termRef.symbol.is(Extension)` condition part not needed anymore?

Reply:
It is filtered in later parts, but I'll restore it just to align it with the variable